controllers/subscription/subscription_controller.go

```diff
@@ -186,16 +188,26 @@ func (r *SubscriptionReconciler) HandleUpgrades(ctx context.Context, rhmiSubscri
 
 		return ctrl.Result{}, nil
 	}
+	log.Infof("RHMI subscription", l.Fields{"StartingCSV": rhmiSubscription.Spec.StartingCSV, "InstallPlanRef": rhmiSubscription.Status.InstallPlanRef})
+	latestRHMIInstallPlan := &olmv1alpha1.InstallPlan{}
+	err := wait.Poll(time.Second*5, time.Minute*5, func() (done bool, err error) {
+		// gets the subscription with the recreated installplan
+		err = r.Client.Get(ctx, k8sclient.ObjectKey{Name: rhmiSubscription.Name, Namespace: rhmiSubscription.Namespace}, rhmiSubscription)
+		if err != nil {
+			log.Info("Couldn't get subscription")
```

Comment:
same here, couldn't retrieve the subscription and add the error to the log
```suggestion
log.Infof("Couldn't retrieve the subscription due to an error ", l.Fields{"Error": err})
```